csm/core/blogic/const.py

```diff
@@ -999,3 +1002,16 @@
 ERROR_CODE = "error_code"
 MESSAGE_ID = "message_id"
 JSON_ERROR = "Unable to parse request body, Malformed request body received."
+
+# Query deployment
+VALID_RESOURCES = ['cluster']
+VALID_VIEWS = ['version', 'nodes', 'storage_set', 'certificate']
```

Comment:
Valid is not looking good. Also resources is too generic term.

Reply:
updated name